services/ui-src/src/utils/other/export.tsx

```diff
@@ -109,6 +119,17 @@ export const renderDrawerDataCell = (
               index
             )}
           </li>
+          {/* If there are nested ILOS responses available, render them here */}
+          {nestedIlosResponses &&
```

Comment:
If you have `let nestedIlosResponses = [];`, you can remove `nestedIlosResponses &&`.